products/chef-infra-server.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Chef Infra Server
+category: server-app
+iconSlug: chef
+tags: erlang-runtime ruby-runtime
+permalink: /chefinfraserver
+alternate_urls:
+-   /chefserver
+releasePolicyLink: "https://docs.chef.io/versions/"
+changelogTemplate: "https://docs.chef.io/release_notes_server/#__LATEST__/"
+versionCommand: chef-server-ctl version
+releaseDateColumn: true
+activeSupportColumn: true
+
+auto:
+-   git: https://github.com/chef/chef-server
+    # https://regex101.com/r/ZBVO7U/1
+    regex: ^(?<major>\d+)\.?(?<minor>\d{1,3})\.?(?<patch>\d+)\.?(?<tiny>\d+)?$
+
+# eol(x) = releaseDate(x+2) > confirm EOL date here https://docs.chef.io/versions/
+# support(x) = releaseDate(x+1)
+releases:
+# -   releaseCycle: "16"
+#    releaseDate: 2023-11-26
+#    support: true
+#    eol: false
+#    latest: "16.0.0"
+#    latestReleaseDate: 2023-11-26
```

Comment:
v16 is available on github but I haven't found any announcement anywhere.
And it's not even on the official page, I think the best for now is to leave it commented

I don't know what to think about it :/
Apart from that, LGTM

Reply:
Chef uses a tool called Expeditor that they wrote which handles major version bumping. It looks like November someone accidently applied the GitHub label to a PR which bumpbs the major version. They then reverted it in a followup commit. The versions in the VERSION file on main are not the actual released versions. You need to check what ends up on the docs site or what goes into the discourse. 16.0 should get removed here. It's probably going to be a while before they do a major version bump.  They did release a new minor version today though so this PR should get updated for that:

https://discourse.chef.io/t/chef-infra-server-15-9-20-released/22743

Reply:
removed & updated, ty